apps/system/js/media_playback.js

```diff
@@ -155,19 +155,30 @@ MediaPlaybackWidget.prototype = {
     }
 
     var title = metadata.title.trim();
+    var titleNode = document.createTextNode(title || '');
+    var titleBdiNode = document.createElement('bdi');
+    titleBdiNode.appendChild(titleNode);
+
     var artist = metadata.artist.trim();
-    var track = [];
+    var artistNode = document.createTextNode(artist || '');
+    var artistBdiNode = document.createElement('bdi');
+    artistBdiNode.appendChild(artistNode);
```

Comment:
Please move this new code that creates the text nodes and bdi elements down below inside the `if (title)` and `if (artist)` blocks. As it stands, you're always creating these things even if they are not used.